### PR TITLE
Solve js3/1.js

### PR DESCRIPTION
**Pull request by achentha**

### CODE REVIEW

File: curriculum/js3/1.js [hunk 1]
```diff
@@ -4,11 +4,30 @@
...
 const solution = (obj)=>{
-  return 0;
+  let val = Object.values(obj);
+  if (val.length === 0) return null;
+  return val.reduce((max,el) => el>max? el : max);
+}
+*/
+
+/* ver 2 - recursive */
+const findMax = (arr,max) => {
```
Comment (songz) [inline curriculum/js3/1.js]:
```
const findMax = (arr, max=arr[0], i=1) => {
  if( i >= arr.length) return max;
  return findMax(arr, arr[i] > max ? arr[i] : max, i+1);
}
```

File: curriculum/js3/1.js [hunk 2]
```diff
@@ -4,11 +4,45 @@
...
+*/
+
+/* ver 3 - using index i instead of slice() */
+const findMax = (arr, max=arr[0], i=1) => {
+  if (i >= arr.length) return max;
+  return findMax(arr, arr[i] > max? arr[i] : max, i+1);
+}
+
+const solution = (obj) => {
+  let val = Object.values(obj);
```
Comment (songz) [inline curriculum/js3/1.js]: const

File: curriculum/js3/1.js [hunk 3]
```diff
@@ -4,11 +4,45 @@
...
+  if (!val.length) return null;
+  else if (val.length === 1) return val[0];
```
Comment (songz) [inline curriculum/js3/1.js]: if and else if is not necessary.

**Comment by pxa4576** [comment]
hey chen, whats ur userid on gs slack channel?

**Comment by achentha** [comment]
I don't think I am in gs slack channel yet. I was not even aware of it. Should I go ahead and add myself in there? What is the exact name of the channel?
thankschen
Apichai (Chen) Chenthanakij
LinkedIn | GitHub | GA
 

    On Thursday, May 17, 2018, 4:42:07 PM PDT, pxa4576 <notifications@github.com> wrote:  
 
 
hey chen, whats ur userid on gs slack channel?

**Comment by achentha** [comment]
I just tried to add myself into garagescript.slack.com . It failed both with my badchen@yahoo.com and badchen@garagescript.org .Can you please advise me what I should use?
thankschen
Apichai (Chen) Chenthanakij
LinkedIn | GitHub | GA
 

    On Thursday, May 17, 2018, 4:42:07 PM PDT, pxa4576 <notifications@github.com> wrote:  
 
 
hey chen, whats ur userid on gs slack channel?

**Comment by pxa4576** [comment]
i have messaged Song about it, please ask him when u see him. I will message u there

**Pull request merged**